source/MapPanel.cpp

```diff
@@ -231,7 +231,7 @@ namespace {
 	}
 
 	// Return total value of raid fleet (if any) and 60 frames worth of system danger.
-	double DangerFleetTotal(const PlayerInfo &player, const System &system, const bool withRaids)
+	double DangerFleetTotal(PlayerInfo &player, const System &system, const bool withRaids)
```

Comment:
I don't like this. While this solution is simple, I'm afraid it has potential to introduce bugs that would be really hard to track down.
So, instead I suggest to split `RaidFleetAttraction` into two functions: one `const` that calculates the value (used here) and another one that takes that value and does the updates that require it to be non-`const`.

Reply:
I moved the thing I wanted to modify in the namespace so we don't need a const playerinfo and I don't need to calculate things twice and possibly break the logic I half forgot ha

Reply:
If you want to go this way, `mutable` would be better. But technically it's still possible to modify that field by just opening the map...

Reply:
Except we won't modify it like that?
I'm not sure where the issue is actually

Reply:
ok I guess we do similar things in a few other places, so just do mutable